client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -308,18 +387,18 @@ class ListItem extends PureComponent {
               <span className="list-item__author">{`Added by: ${authorName}`}</span>
             </span>
           </label>
-          {!isOrdered && (
-            <VotingBox
-              isVoted={isVoted}
-              voteForItem={voteForItem}
-              votesCount={votesCount}
-            />
-          )}
+          {this.renderVoting()}
           <button
```

Comment:
maybe create component `PendingButton`? Or something similar as the same code is copied multiple times here.